### DISCUSSION TITLE
MM2 Replicate only latest offsets

### DISCUSSION THREAD

**Discussion by Sharad-SKJ052** [strimzi/strimzi-kafka-operator#9588] [Q&A] [answered]
Hi,

I'm running a multi-region Kafka setup using strimzi operator version 0.35.1 and kafka running on 3.4.0. The replication process that I would like to implement is for a pre-existing topic in source cluster, MM2 should replicate the events after the replication is established. 

For Example - 

Topic A is Source cluster has 15 events. After the topic is whitelisted, it should not replicate the existing events and should in turn start from 16th message once it's produced to Topic A. 

Will this kind of scenario be supported with MM2? If so, can you please let me know what parameters are crucial to replicate the latest offsets only.

**Comment by scholzj** [comment]
I think there is a Kafka issue for this: https://issues.apache.org/jira/browse/KAFKA-13988. So you need to wait for it to be released in Apache Kafka first.

**Reply by Sharad-SKJ052** [reply]
@scholzj  Can we sync the consumer offset via MM2 without syncing the events.

**Reply by scholzj** [reply, marked as answer]
I don't think so. The offsets are translated based on how the topic records are synced. So without having the new offsets for the records, it is not possible to translate the consumer offsets. But it is more a Mirror Maker 2 than a Strimzi question, so I'm not 100% sure.

**Reply by Sharad-SKJ052** [reply]
Sure thanks
